debian/patches/ds-identify-behavior-xenial.patch

```diff
@@ -10,18 +10,18 @@ Bug-ubuntu: http://bugs.launchpad.net/bugs/1660385
 
 --- a/tools/ds-identify
 +++ b/tools/ds-identify
-@@ -93,8 +93,8 @@ _DI_LOGGED=""
- DI_MAIN=${DI_MAIN:-main}
+@@ -94,8 +94,8 @@ DI_MAIN=${DI_MAIN:-main}
  
- DI_BLKID_OUTPUT=""
+ DI_BLKID_EXPORT_OUT=""
+ DI_GEOM_LABEL_STATUS_OUT=""
 -DI_DEFAULT_POLICY="search,found=all,maybe=all,notfound=${DI_DISABLED}"
 -DI_DEFAULT_POLICY_NO_DMI="search,found=all,maybe=all,notfound=${DI_ENABLED}"
-+DI_DEFAULT_POLICY="report,found=all,maybe=all,notfound=${DI_ENABLED}"
++DI_DEFAULT_POLICY="report,found=all,maybe=all,notfound=${DI_DISABLED}"
```

Comment:
This doesn't look right to me; the diff we needed to fix daily was ed5471f1cce33ef17a13321cb6436a2b4dc9973d:

```diff
--- a/debian/patches/ds-identify-behavior-xenial.patch
+++ b/debian/patches/ds-identify-behavior-xenial.patch
@@ -10,10 +10,10 @@ Bug-ubuntu: http://bugs.launchpad.net/bugs/1660385
 
 --- a/tools/ds-identify
 +++ b/tools/ds-identify
-@@ -93,8 +93,8 @@ _DI_LOGGED=""
- DI_MAIN=${DI_MAIN:-main}
+@@ -94,8 +94,8 @@
  
- DI_BLKID_OUTPUT=""
+ DI_BLKID_EXPORT_OUT=""
+ DI_GEOM_LABEL_STATUS_OUT=""
 -DI_DEFAULT_POLICY="search,found=all,maybe=all,notfound=${DI_DISABLED}"
 -DI_DEFAULT_POLICY_NO_DMI="search,found=all,maybe=all,notfound=${DI_ENABLED}"
 +DI_DEFAULT_POLICY="report,found=all,maybe=all,notfound=${DI_ENABLED}"
@@ -21,7 +21,7 @@ Bug-ubuntu: http://bugs.launchpad.net/bugs/1660385
  DI_DMI_CHASSIS_ASSET_TAG=""
  DI_DMI_PRODUCT_NAME=""
  DI_DMI_SYS_VENDOR=""
-@@ -131,7 +131,7 @@ DI_ON_FOUND=""
+@@ -132,7 +132,7 @@
  DI_ON_MAYBE=""
  DI_ON_NOTFOUND=""
```

Reply:
@OddBloke I can't seem to find that commitish you mentioned anywhere.  Did you mean the commit in master cf6c36a18de233190996f7c2dde1e04908bad50a? which introduced part of that change to ds-identify?

Reply:
I meant https://github.com/canonical/cloud-init/commit/dd17691cc24a3b69b67b56078e147bca3e283ae6; evidently I pushed and then somehow changed my local history!

Reply:
> This doesn't look right to me; the diff we needed to fix daily was ed5471f1cce33ef17a13321cb6436a2b4dc9973d
Thanks a lot for catching that. my diff now against upstream/ubuntu/xenial is now


```diff
ubuntu/xenial)$ git diff origin/ubuntu/xenial  debian/patches/ds-identify-behavior-xenial.patch 
diff --git a/debian/patches/ds-identify-behavior-xenial.patch b/debian/patches/ds-identify-behavior-xenial.patch
index ba7639ab..25cc3813 100644
--- a/debian/patches/ds-identify-behavior-xenial.patch
+++ b/debian/patches/ds-identify-behavior-xenial.patch
@@ -10,10 +10,10 @@ Bug-ubuntu: http://bugs.launchpad.net/bugs/1660385
 
 --- a/tools/ds-identify
 +++ b/tools/ds-identify
-@@ -93,8 +93,8 @@ _DI_LOGGED=""
- DI_MAIN=${DI_MAIN:-main}
+@@ -94,8 +94,8 @@ DI_MAIN=${DI_MAIN:-main}
  
- DI_BLKID_OUTPUT=""
+ DI_BLKID_EXPORT_OUT=""
+ DI_GEOM_LABEL_STATUS_OUT=""
 -DI_DEFAULT_POLICY="search,found=all,maybe=all,notfound=${DI_DISABLED}"
 -DI_DEFAULT_POLICY_NO_DMI="search,found=all,maybe=all,notfound=${DI_ENABLED}"
 +DI_DEFAULT_POLICY="report,found=all,maybe=all,notfound=${DI_ENABLED}"
@@ -21,7 +21,7 @@ Bug-ubuntu: http://bugs.launchpad.net/bugs/1660385
  DI_DMI_CHASSIS_ASSET_TAG=""
  DI_DMI_PRODUCT_NAME=""
  DI_DMI_SYS_VENDOR=""
-@@ -131,7 +131,7 @@ DI_ON_FOUND=""
+@@ -132,7 +132,7 @@ DI_ON_FOUND=""
  DI_ON_MAYBE=""
  DI_ON_NOTFOUND=""
```